src/main/kotlin/com/autonomousapps/internal/JarExploder.kt

```diff
@@ -43,7 +43,7 @@ internal class JarExploder(
   private fun explodeJar(artifact: PhysicalArtifact): ExplodingJar {
     val zip = ZipFile(artifact.file)
 
-    val alreadyExplodingJar: ExplodingJar? = inMemoryCache.explodedJar(artifact.coordinates.toString())
+    val alreadyExplodingJar: ExplodingJar? = inMemoryCache.explodedJar(artifact.toString())
```

Comment:
Can you explain this change?

Reply:
I think I misunderstood parts of the code here. This change has no effect as it is.

But I think, the previous code is also wrong. Later we have this:

https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/blob/b7a7fbc71b15b6cc7277dd8402a99ab66b5abc24/src/main/kotlin/com/autonomousapps/internal/JarExploder.kt#L76

So shouldn't this be `artifact.file.name`? I think right now (with or without my change) `inMemoryCache.explodedJar(...)` always returns `null`.

Using just the "file name" as key might also not be enough though. Maybe both places should be changed to use `absolutePath`. Wdyt?

(I made this change now, but can revert if I misunderstood the intentions.)

Reply:
https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/pull/688/commits/0083fddc453f82e15adc096d4f104d5ee0a36feb